package.json

```diff
@@ -45,20 +45,26 @@
   },
   "homepage": "https://github.com/bcherny/json-schema-to-typescript#readme",
   "dependencies": {
+    "@types/is-glob": "^4.0.1",
     "@types/json-schema": "^7.0.3",
-    "@types/node": ">=4.5.0",
+    "@types/mkdirp": "^0.5.2",
+    "@types/node": "^13.1.2",
```

Comment:
Any chance we could make this a bit more permissive, so it works with peoples' existing `@types/node`? eg. `>=10` or `>=11` would be nice, since 12 is LTS.